### ISSUE TITLE
the value-title part of the value-class pattern isn't implemented

### ISSUE THREAD

**Issue by kevinmarks** [tommorris/mf2py#37]
for example, see https://github.com/microformats/tests/blob/master/tests/microformats-v2/h-geo/hidden.html

`````
<p>
    <span class="h-geo">The Bricklayer's Arms
        <span class="p-latitude">
            <span class="value-title" title="51.513458"> </span> 
        </span>
        <span class="p-longitude">
            <span class="value-title" title="-0.14812"> </span>
        </span>
    </span>
</p>
`````

should give
`````
{
    "items": [{
        "type": ["h-geo"],
        "properties": {
            "latitude": ["51.513458"],
            "longitude": ["-0.14812"],
            "name": ["The Bricklayer's Arms"]
        }
    }],
    "rels": {}
}
`````

but we don't get the lat/long

**Comment by kylewm** [comment]
implemented here https://github.com/kylewm/mf2py/commit/b98a950ec396658dd2ba0664cfb8ced651aecf22

**Comment by kylewm** [comment]
fixed in b81a950b39ead178607def82ee59815f3f425fd8
